source/vocab/relations.ttl

```diff
@@ -189,6 +189,28 @@
     rdfs:subClassOf :Document;
     owl:equivalentClass sdo:MediaObject .
 
+:image a owl:ObjectProperty;
+    rdfs:label "image"@en, "bild"@sv;
+    rdfs:comment "En bild på resursen"@sv;
+    rdfs:domain :Resource;
+    rdfs:range :ImageObject;
+    rdfs:subPropertyOf :isPrimaryTopicOf;
+    owl:equivalentProperty sdo:image .
+
+:thumbnail a owl:ObjectProperty;
+    rdfs:label "thumbnail"@en, "miniatyrbild"@sv;
+    skos:altLabel "tumnagel"@sv, "indexbild"@sv;
+    rdfs:comment "En förminskad bild"@sv;
+    sdo:domainIncludes :ImageObject;
+    sdo:rangeIncludes :ImageObject;
+    rdfs:subPropertyOf :image;
```

Comment:
Is this *between these* to relationships? Neither just an image of the resource, nor just another format thereof; but both.

```suggestion
    rdfs:subPropertyOf :image, :hasFormat ;
```

Reply:
... But that (`hasFormat`) doesn't hold for a thumbnail of a video! `hasVersion` sort of works. I'm fine if we reject this for now. (Would be good to note this fact in a comment in the ttl though.)